scss/_settings_colors.scss

```diff
@@ -91,12 +91,9 @@ $colors--light-theme--border-low-contrast: rgba($color-x-dark, 0.1) !default;
 
 // Dark theme
 $colors--dark-theme--text-default: hsl(0, 0%, 100%) !default;
-$colors--dark-theme--nav-link-hover: $color-x-light !default;
-$colors--dark-theme--text-hover: hsl(0, 0%, 76%) !default; // minimum contrast on primary that satisfies contrast checker AA
 $colors--dark-theme--text-disabled: rgba($colors--dark-theme--text-default, 0.55) !default;
 $colors--dark-theme--text-muted: rgba($colors--dark-theme--text-default, $muted-text-opacity-amount) !default;
 $colors--dark-theme--text-inactive: rgba($colors--dark-theme--text-default, $inactive-text-opacity-amount) !default;
-$colors--dark-theme--nav-link-hover: $colors--dark-theme--text-hover !default;
```

Comment:
This doesn't seem related to spacing. I guess it's just a drive-by clean up - are those not used anywhere? (I see that we don't have those in light theme).

Reply:
nowhere in org:canonical-web-and-design

Reply:
@bartaz ready for another look.